LinuxPackaging.cmake

```diff
@@ -0,0 +1,95 @@
+# support creating some basic binpkgs via `make package`
+set(CPACK_GENERATOR TGZ;RPM;DEB)
+
+set(CPACK_INCLUDE_TOPLEVEL_DIRECTORY 0)
+set(CPACK_PACKAGE_VERSION_MAJOR ${LIBIIO_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${LIBIIO_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH g${LIBIIO_VERSION_GIT})
+set(CPACK_BUNDLE_NAME libiio)
+set(CPACK_PACKAGE_VERSION ${LIBIIO_VERSION})
+# debian specific package settings
+set(CPACK_PACKAGE_CONTACT
+       "Engineerzone <https://ez.analog.com/community/linux-device-drivers>")
+
+FIND_PROGRAM(DPKG_CMD dpkg)
+FIND_PROGRAM(DPKGQ_CMD dpkg-query)
```

Comment:
OK - I was thinking that if if was not set, or if it could not find the dpkg, it would just use defaults, not actually when someone says "make package". But I can do it that way too..